yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/command/QueryConfig.java

```diff
@@ -361,6 +361,23 @@ void checkResultInternal(@Nonnull final Object actual, @Nonnull final String que
         };
     }
 
+    /**
+     * Return a NoOp config - a config that does nothing.
+     * This config can be executed but will perform no action. Use in cases where we need to continue running (e.g.
+     * the command and config are legal and supprted) but some conditions make execution unneeded.
+     * @param lineNumber the line number in the test file
+     * @param executionContext the execution context for the test
+     * @return an instance of a NoOp config
+     */
+    public static QueryConfig getNoOpConfig(int lineNumber, @Nonnull YamlExecutionContext executionContext) {
+        return new QueryConfig(QUERY_CONFIG_NO_OP, null, lineNumber, executionContext) {
+            @SuppressWarnings("PMD.CloseResource") // lifetime of autocloseable persists beyond method
+            @Override
+            void checkResultInternal(@Nonnull Object actual, @Nonnull String queryDescription) throws SQLException {
+            }
```

Comment:
minor: Add assertion about it not executing

Reply:
Sure, will do in a subsequent PR.